rfc-012-makefile-standards.md

```diff
@@ -0,0 +1,64 @@
+# Makefile standards
+
+## Summary
+
+A set of standards for Makefiles across all projects
+
+## Problem
+
+Having a standard set of commands that can be run across all projects for common tasks
+allows people to get projects up and running without having to spend time looking into
+how things are set up to know what specific commands to run.
+
+
+## Proposal
+
+All projects MUST have a Makefile.
+
+All projects must have relevant [`PHONY` targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)
+
+**All** projects MUST have the commands:
+- `make test`
```

Comment:
Suggest test commands could have optional args for individual tests/files where supported, though this is more of a nice to have and doesn't have to be in the RFC.
It might be nice to include in the example makefile though!

Reply:
Yeah that might be good, maybe as a `MAY`